Bot fails to start when using github.com/slack-go/slack v0.13.0

When go: upgraded github.com/slack-go/slack v0.12.2 => v0.13.0 bot fails to start with the following error:

`# github.com/slack-io/slacker
../../go/pkg/mod/github.com/slack-io/slacker@v0.1.0/slacker.go:547:40: cannot use messageEvent.BotID (variable of type string) as slack.GetBotInfoParameters value in argument to s.slackClient.GetBotInfo`

--- a/go.mod
+++ b/go.mod
@@ -6,10 +6,10 @@ require (
 	github.com/robfig/cron/v3 v3.0.1
 	github.com/shomali11/commander v0.0.0-20230730023802-0b64f620037d
 	github.com/shomali11/proper v0.0.0-20190608032528-6e70a05688e7
-	github.com/slack-go/slack v0.12.2
+	github.com/slack-go/slack v0.13.0
 )
 
 require (
-	github.com/gorilla/websocket v1.5.0 // indirect
+	github.com/gorilla/websocket v1.5.3 // indirect
 	github.com/stretchr/testify v1.8.4 // indirect
 )

--- a/slacker.go
+++ b/slacker.go
@@ -544,7 +544,7 @@ func (s *Slacker) ignoreBotMessage(messageEvent *MessageEvent) bool {
 		if messageEvent.BotID == slackBotID {
 			return false
 		}
-		bot, err := s.slackClient.GetBotInfo(messageEvent.BotID)
+		bot, err := s.slackClient.GetBotInfo(slack.GetBotInfoParameters{Bot: messageEvent.BotID})
 		if err != nil {
 			if err.Error() == "missing_scope" {
 				s.logger.Error("unable to determine if bot response is from me -- please add users:read scope to your app")

